src/sphinx_book_theme/assets/styles/content/_margin.scss

```diff
@@ -38,6 +38,48 @@ $content-fullwidth-width: percentage(100% / $content-max-width);
   }
 }
 
+label.margin-toggle {
```

Comment:
Can we add a `user-select: none` somewhere for the superscript number?

https://css-tricks.com/almanac/properties/u/user-select/

Reply:
Added here https://github.com/executablebooks/sphinx-book-theme/pull/546/commits/75d40af99d42133cd7d5c9fd70a14518bf2eea20